api/flask_api.py

```diff
@@ -0,0 +1,27 @@
+import flask_restful
+import responses
+import requests
+from flask import Flask
+from flask_restful import Resource, Api, reqparse
+import pandas as pd
+import ast
+
+from flask_cors import CORS, cross_origin
+
+app = Flask(__name__)
+api = flask_restful.Api(app)
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+class email(Resource):
+    def post(self):
+        return {'message': "posted successfully"}, 200
+
```

Comment:
I require changes. Remove this empty line here

Reply:
I refuse. down with the patch-riarchy